docs/developer-onboarding.md

```diff
@@ -14,3 +20,34 @@ PaaS is built on CloudFoundary and we use the CloudFoundary CLI to interact with
 
 ### Getting familiar with PaaS
 See [PaaS cheatsheet](/docs/paas-cheatsheet.md).
+
+
+### Azure CIP
+
+You will need a CIP account in order to raise a PIM (Privileged Identity Management) request to access certain resources, for example, production environments. Steps on how to register for a CIP account and raise a PIM request
+can be found in the [DfE Technical Guidance](https://technical-guidance.education.gov.uk/infrastructure/hosting/azure-cip/).
+
+## Access to QA, Sandbox and Production
+
+- Navigate to the Support Sign in page: [QA](https://qa.apply-for-teacher-training.service.gov.uk/support/sign-in), [Staging](https://staging.apply-for-teacher-training.service.gov.uk/support/sign-in) and [production](https://www.apply-for-teacher-training.service.gov.uk/support/sign-in)
+- If you already have a DfE Sign In account, login, if you do not click `Create account` and follow the instructions
+- Once you have successfully logged in, you should see an [Apply page](https://qa.apply-for-teacher-training.service.gov.uk/support) stating your `UUID`
+- Ask a colleague who has support access for the environment you are trying to access to go to the [Add support user page](https://www.apply-for-teacher-training.service.gov.uk/support/users/support/new) and add your email address and your `UUID` (from the last step).
+
+Note: You can also navigate to the Add Support User page by logging into support and clicking `Settings`, `Support users` and `Add support user`
+
+- You should now be able to login.
+
+## Developer Slack Channels
+These are a small number of recommended Slack channels for new developers to join:
+- `#apply-dev-notifications`
+- `#civil_servant_devs`
```

Comment:
Do you think it's worth adding a note here explaining this is a locked channel, they won't be able to search for it and they will need to be invited by someone who already has access?

Reply:
Yes – I did wonder about this and made me think if this channel still needed to be locked 🤔. A discussion for another time perhaps, but happy with a note being added – guess they will just need a way of identifying who to ask i.e. which devs are CS within their team

Reply:
I think when Aga created this channel, we thought it might encourage conversation etc. i think there are pros and cons on both sides, but yes, you're right, a conversation for another day. 

I'll add a bit about it to my commit.